src/app/project/project.component.html

```diff
@@ -103,8 +103,26 @@ <h1>Find EA Projects in British Columbia</h1>
             <option label="Post-Certification" value="Post-Certification">Post-Certification</option>
           </select>
         </div>
+
+        <div class="col-md-2">
+					<label>Region</label>
+
+					<select class="form-control" [(ngModel)]="regionFilter" (ngModelChange)="config.currentPage = 1">
```

Comment:
Please alphabetize the content of the dropdown.
Is there any way to have the list of proponents be dynamically loaded, rather than hard-coded?

Reply:
Just to avoid confusion, you meant _list of regions_, right?

Reply:
Yes, I was got those two crossed :)